README.md

```diff
@@ -154,8 +154,8 @@ Awesome Wagtail [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38
 
 ### Forms
 
-- [wagtailpolls](https://github.com/takeflight/wagtailpolls) - A plugin for adding polling capabilities to the Wagtail CMS.
-- [Wagtailsurveys](https://github.com/torchbox/wagtailsurveys) - A module for Wagtail which provides the ability to build polls and surveys.
+Before looking into form add-ons, take a look at [Wagtail's Form Builder](https://docs.wagtail.org/en/stable/reference/contrib/forms/).
```

Comment:
Just a thought - maybe we can keep these as a good reference but add a label of some kind like 'archived' or 'unmaintained'

Reply:
It's up to the maintainers of course, but here's my 2 cents:

As a user of Awesome-<anything>, i'm an "integrator" kind of developer, looking for an add-on that "Just Works (TM)". So seeing old stuff would only detract from my experience. 

If i were a developer looking for coding examples, i'd assume that something built for a 5 year old version of the framework would not be applicable anymore.

Reply:
yeah, I think I agree with you.
If these packages do end up being cloned/updated in the future, someone can simply re-add them here.